src/ios/VLCPlugin.m

```diff
@@ -325,48 +373,70 @@ - (void)playremotefile:(CDVInvokedUrlCommand*)command {
         pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_ERROR messageAsString:@"invalid remote file url"];
     }
     
-    [self vlc_sendPluginResult:pluginResult callbackId:command.callbackId];
+    [self vlc_sendPluginResult:pluginResult callbackId:self.callbackId];
 }
 
 - (void)pause:(CDVInvokedUrlCommand*)command {
+    [self pauseWithCallbackId:command.callbackId];
+}
+
+- (void)stop:(CDVInvokedUrlCommand*)command {
+    [self stopPlayback];
+}
+
+- (void)setaudioinfo:(CDVInvokedUrlCommand*)command{
+    NSDictionary  * info = [command.arguments  objectAtIndex:0];
+    [self vlc_setlockscreenmetadata:info refreshLockScreen:true];
+    
+    if ([self.audio isKindOfClass:[NYPRStream class]] && [info objectForKey:@"svg_logo"]) {
```

Comment:
This method is not called a lot and it is cleanest to create a new show when the show in the live stream changes.